pkg/apis/flow/v1alpha1/jobflow_types.go

```diff
@@ -35,6 +35,11 @@ type JobFlowSpec struct {
 	Flows []Flow `json:"flows,omitempty"`
 	// +optional
 	JobRetainPolicy RetainPolicy `json:"jobRetainPolicy,omitempty"`
+	// Defaults to 3.
```

Comment:
The design document states 1 but the default here is 3.

Reply:
Thanks @googs1025 
Fixed. Change to default value 1